Doc/tools/extensions/suspicious.py

```diff
@@ -115,10 +115,12 @@ def write_doc(self, docname, doctree):
     def finish(self):
         unused_rules = [rule for rule in self.rules if not rule.used]
         if unused_rules:
-            self.warn('Found %s/%s unused rules:' %
```

Comment:
Does similar change of using `self.logger.warning` also need to be done in `report_issue` method?

* https://github.com/python/cpython/blob/aaf47caf35984e614d93bd8bea5227df55e0e3e6/Doc/tools/extensions/suspicious.py#L154
* https://github.com/python/cpython/blob/aaf47caf35984e614d93bd8bea5227df55e0e3e6/Doc/tools/extensions/suspicious.py#L157

Reply:
probably!  let me fix those too -- thanks for pointing them out 👍